libr/util/str.c

```diff
@@ -672,7 +672,10 @@ R_API char *r_str_trunc_ellipsis(const char *str, int len) {
 	if (strlen (str) < len) {
 		return strdup (str);
 	}
-	char *buf = r_str_newlen (str, len);
```

Comment:
r_str_newlen is deprecated too, right?

Reply:
yes

Reply:
maybe r_str_ndup should handle negative length in the same way or we shuol dhave a macro for that, as it seems the alternative is made in so many different places